### DISCUSSION TITLE
Custom Register Validation

### DISCUSSION THREAD

**Discussion by kerorojason** [react-hook-form/react-hook-form#2264] [Q&A] [answered]
Hi all.
When I use custom register, my validation doesn't work! I've be debugging for over a week. Would appreciate your help!
https://codesandbox.io/s/trusting-mahavira-piq9p?from-embed

**Comment by bluebill1049** [comment, marked as answer]
it's important to leave steps to reproduce.
anyway i try to guess:

`setValue("custom_field", e.target.value, true);`
